src/gui-wizard-gtk/wizard.c

```diff
@@ -124,8 +124,12 @@ static GtkExpander *g_exp_search;
 static gulong g_tv_sensitive_sel_hndlr;
 static gboolean g_warning_issued;
 
+static GtkWidget *report_stack;
+static GtkWidget *report_status_box;
+static GtkWidget *report_warning_box;
 static GtkSpinner *g_spinner_event_log;
 static GtkImage *g_img_process_fail;
+static GtkWidget *report_warning_label_box;
```

Comment:
Can you rename the vars so they follow the same pattern as other vars do? `g_` + `name_of_the_variable`